### DISCUSSION TITLE
[BUG] плод сделался на 100 процентов пишет не определенно после пишет синхронизация после пишет фарминг тоесть процесс завершен а после через какое то время опять не определенно и пишет дождитесь синхронизации

### DISCUSSION THREAD

**Discussion by ceregaaaaa** [Chia-Network/chia-blockchain#3263] [Support]
плод сделался на 100 процентов пишет не определенно после пишет синхронизация после пишет фарминг тоесть процесс завершен а после через какое то время опять не определенно и пишет дождитесь синхронизации 
![6MX7fRuOH3o](https://user-images.githubusercontent.com/83298122/116273376-5cd94380-a78a-11eb-92ed-741f36d93a3d.jpg)
![CBNKwBzWPAw](https://user-images.githubusercontent.com/83298122/116273409-6367bb00-a78a-11eb-91b3-dfd679c5e4fe.jpg)

**Comment by Crypto-78** [comment]
you are really smart to write in Russian language, everyone knows Russian 🤣🤣🤣🤣🤣🤣

**Comment by f1sc4ll** [comment]
@ceregaaaaa Изображения мало говорят, нажмите на меню из трех пунктов справа, у вас есть доступ к журналу отладки...скопируйте и вставьте журнал сюда.

**Comment by ceregaaaaa** [comment]
@f1sc4ll  я бы с радостью но там нет пункта журнал потому что процесс зас5еивания уже завершился и есть пункт только удалить

**Comment by f1sc4ll** [comment]
@ceregaaaaa  Понятно, поэтому вам нужно попробовать еще раз и сохранить журнал.

Здесь я тестирую различные конфигурации и теряю «10 plots K32» ... лучший способ, который я нашел до сих пор, заключался в том, чтобы иметь компьютер для создания «farm» и еще один для фермы.

**Comment by aqk** [comment]
спасибо @f1sc4ll 

@ceregaaaaa

Ваш сюжет в порядке.

Однако, если вы нажмете на «полный узел», вы можете увидеть «Статус: не синхронизирован».

Этот желтый цвет говорит о том, что с плодами все в порядке, но они еще не обмениваются данными с сетью.

Я попрошу нас обновить этот текст, но это займет некоторое время.

![not-synced](https://user-images.githubusercontent.com/338792/116328615-42996880-a77e-11eb-963d-aab0a2a33b5c.png)

**Comment by ZOMboy163** [comment]
Скажите пожалуйста, у меня у одного зависла дата пикового время и синхронизация дальше не идёт?
От чего зависит эта синхронизация?

**Comment by stalit1** [comment]
Попробуй в ручную удалить временую папку для засева. У меня было что весь диск M2 засрался  всё встало. Удалил мусор на 800 GB всё заработало.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
